autoblocks/_impl/tracer.py

```diff
@@ -101,6 +109,60 @@ def start_span(self):
                 props["span_id"] = prev_span_id
             self.set_properties(props)
 
+    async def evaluate_event(self, event: TracerEvent, evaluator: BaseEventEvaluator) -> None:
```

Comment:
shouldn't this have an underscore in front? it's not meant to be used by an end user right?